### PR TITLE
updated plugin and report dependencies to latest version

### PR DESCRIPTION
**Pull request by serious6**
- versions have also been extracted for easier overview

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@serious6__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

**Comment by vbauer** [comment]
:+1:

File: pom.xml [hunk 1]
```diff
@@ -59,6 +59,28 @@
...
+        <dep.nexus-staging-plugin.version>1.6.5</dep.nexus-staging-plugin.version>
+        <dep.source-plugin.version>2.4</dep.source-plugin.version>
+        <!--  Dependencies [REPORT]:  -->
+        <dep.site-plugin.version>3.4</dep.site-plugin.version>
+        <dep.project-info-plugin.version>2.8</dep.project-info-plugin.version>
+        <dep.versions-plugin.version>2.1</dep.versions-plugin.version>
+        <dep.jxr-plugin.version>2.5</dep.jxr-plugin.version>
+        <dep.surefire-plugin.version>2.18.1</dep.surefire-plugin.version>
+        <!--  Dependencies [COMPILE]:  -->
+        <dep.httpclient.version>4.3.5</dep.httpclient.version>
```
Comment (vbauer) [inline pom.xml]: Last version: 4.4.1

File: pom.xml [hunk 2]
```diff
@@ -59,6 +59,28 @@
...
+        <dep.httpcore.version>4.3.3</dep.httpcore.version>
```
Comment (vbauer) [inline pom.xml]: Last version: 4.4.1

**Comment by serious6** [comment]
httpclient and httpcore have now also been updated

**Pull request merged**